easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.3.0-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'QuantumESPRESSO'
+version = '5.3.0'
+
+homepage = 'http://www.quantum-espresso.org'
+description = """Quantum ESPRESSO is an integrated suite of Open-Source computer
+ codes for electronic-structure calculations and materials modeling  at the nanoscale.
+ It is based on density-functional theory, plane  waves, and pseudopotentials."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://qe-forge.org/gf/download/frsrelease/204/912/']
+sources = ['espresso-%(version)s.tar.gz']
+checksums = ['6848fcfaeb118587d6be36bd10b7f2c3']
+
+configopts = 'DFLAGS="-D__INTEL -D__MPI -D__PARA -D__DFTI -D__SCALAPACK" '
+configopts += 'FFT_LIBS="$LIBFFT" --with-scalapack=intel '
+configopts += '--prefix=%(installdir)s/bin'
+
+buildopts = 'all'
```

Comment:
we usually build QE 'all in', i.e. with `buildopts = 'all w90 want yambo plumed xspectra' `